index.md

```diff
@@ -143,7 +143,7 @@ are embedded as part of a website.
 **WCAG 2.0**
```

Comment:
I think here we want to say “WCAG” only, too. (I also think those bold paragraphs should maybe be headings.)

Reply:
I think this PR is older than what is currently published, which has these references corrected. Correct me if I'm wrong, but I think this PR will not revert that change.

Reply:
I _think_ you’re right.